website/server/controllers/api-v3/chat.js

```diff
@@ -101,6 +111,22 @@ api.postChat = {
     let lastClientMsg = req.query.previousMsg;
     chatUpdated = lastClientMsg && group.chat && group.chat[0] && group.chat[0].id !== lastClientMsg ? true : false;
 
+    // Check for spam by seeing if the current user has posted too many messages in a short window
```

Comment:
Very simple comment yeah, okay I can look at moving this function into the group model, that should make it more reusable too.